assets/sass/components/idea-hub/_googlesitekit-idea-hub-dashboard-ideas-widget.scss

```diff
@@ -0,0 +1,70 @@
+/**
+ * Idea Hub widget styles.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-plugin {
+
+	.googlesitekit-idea-hub {
+		background-color: $c-idea-hub-background;
+		height: 100%;
+	}
+
+	.googlesitekit-idea-hub__header {
+		padding: 24px 16px 0;
+	}
```

Comment:
These styles seem to be off. As far as I see in the figma design it should be `20px 24px 0`, shouldn't it? 

![Screenshot from 2021-05-14 15-01-54](https://user-images.githubusercontent.com/873049/118268653-52c37e80-b4c6-11eb-8f1e-190a5c3c2bf3.png)

Reply:
@eugene-manuilov If we follow the designs, then the title will not be aligned with titles from other widgets. Check the screenshot below:
Padding top set to `24px` 
![image](https://user-images.githubusercontent.com/31818/118275812-f6695a80-b4d7-11eb-9e56-85400dff0626.png)
compared to padding top set to `20px` as per the designs:
![image](https://user-images.githubusercontent.com/31818/118276012-33355180-b4d8-11eb-9554-5a0e5807c525.png)

You will notice that there is some misalignment.

Reply:
Ah... okay. Got it. Thanks!